doc/Overview/Mvux/ListFeeds.md

```diff
@@ -0,0 +1,77 @@
+---
+uid: Overview.Mvux.ListFeeds
+---
+
+# What are List-Feeds?
+
+The `IListFeed<T>` is like a [Feed](xref:Overview.Mvux.Feeds) which is stateless and keeps no track of changes, but is specialized for handling collections.
+
+In Feeds, each data request from the service, returns one single item, whereas in List-Feed each request returns a collection of items.  
+Its distinctive feature is the ability to use its [Operators](xref:Overview.Mvux.ListFeeds#operators) directly on its items, rather than on the entire collection.
+
+Another important thing to remember is that when the service returns an empty collection, it's treated as an Empty message, and the data axis Option will be `None`, although the result was not `null`.
```

Comment:
@dr1rrb is this correct - I would have assumed that if a service returns an empty collection (ie not null) the data axis would return an Option of Some with an empty collection/array